decidim-admin/lib/decidim/admin/engine.rb

```diff
@@ -13,6 +13,9 @@
 require "jbuilder"
 require "rectify"
 
+require_relative "../../../app/models/decidim/admin/concerns/user_extends"
```

Comment:
I don't think explicitly requiring the concern is necessary if you follow the naming conventions - in this case, naming the concern `Decidim::Admin::Concerns::UserExtends`.

`require_relative` often breaks rails' constant autoloading/reloading. If you *really* need to, better use [`require_dependency`](http://guides.rubyonrails.org/autoloading_and_reloading_constants.html#require-dependency)